### PR TITLE
Feature/main class#5

### PR DESCRIPTION
**Pull request by hpar12**

### CODE REVIEW

File: src/test/java/com/coaching/CalculatorTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	@Test
+	public void testZeroPlusTwo() {
+		int actualValue = 0 + 2;
+		int expectedValue = 2;
+		assertEquals("addition of 0 + 2", expectedValue, actualValue);
+	}
+	
+	@Test
+	public void testTwoPlusThree() {
+		int actualValue = 2 + 3;
```
Comment (fraserc) [inline src/test/java/com/coaching/CalculatorTest.java]: This is wrong!!

Comment (hpar12) [reply src/test/java/com/coaching/CalculatorTest.java]: This seems like an incorrect requirement. Can you please verify this for me?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+target/
```
Comment (fraserc) [inline .gitignore]: Now you have this you should be able to delete the target/classes/com/coaching/App.class and other binaries and everything should be sweet.

File: pom.xml [hunk 3]
```diff
@@ -21,5 +21,10 @@
       <version>3.8.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>log4j</groupId>
```
Comment (fraserc) [inline pom.xml]: This is new, what does it do?

Comment (fraserc) [reply pom.xml]: Interesting, so where might we use it and how? 😉

File: .gitignore (hunk 2 above)
Comment (fraserc) [reply .gitignore]: Excellent question. You can basically delete the entire folder structure under target. There's a hack I'll explain after that but you're onto it.

File: pom.xml (hunk 3 above)
Comment (hpar12) [reply pom.xml]: I will resolve this for now as we can look at the logging exercise in the URLyBird project

**Comment by hpar12** [comment]
Hey Fraser, I don't see the App.class or the AppTest.class in the paths.

What am I missing?

**Comment by fraserc** [comment]
@hpar12 fixed yer conflict. Bit tricky as you had to merge with master and then 'officially' delete the file in the commit.

**Pull request merged**